__tests__/app.test.js

```diff
@@ -197,8 +198,62 @@ describe("/api/reviews/:review_id/comments", () =>{
                 expect(body.msg).toBe("404 not found")
             })
         })
+        
 })
-    
+describe("POST /api/reviews/:review_id/comments", () =>{
+    test("when a new comment is posted, its added to the db and the new comment is returned from the db", () =>{
+
+        const newComment = {username : "dav3rid", 
+        body : "What a game changer !!!!!"}
+        return request(app)
+        .post("/api/reviews/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({body}) =>{
+            expect(body.comment).toEqual({author : "dav3rid", 
+                                        body : "What a game changer !!!!!", 
+                                        comment_id : 7, 
+                                        review_id : 1, 
+                                        votes : 0,
+                                        created_at : expect.any(String)})
+        })
+    })
+    test("Query with wrong format (Not Number) ", () => {
+        const newComment = {username : "dav3rid", 
+        body : "What a game changer !!!!!"}
+        return request(app)
+        .post("/api/reviews/one/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe("Bad Request")         
+        }) 
+    })
+    test("404 status and message '404 not found' when pass an incorrect URL", () =>{
+        const newComment = {username : "dav3rid", 
+        body : "What a game changer !!!!!"}
+        return request(app)
+        .post("/api/reviews/2/comment")
+        .send(newComment)
+        .expect(404)
+        .then(({body})=>{
+            expect(body.msg).toBe("404 not found")
+        })
+    }) 
+    test("if passed a user not in the user table", () =>{
+        const newComment = {username : "Rick", 
+        body : "What a game changer !!!!!"}
+        return request(app)
+        .post("/api/reviews/2/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({body})=>{
+            expect(body.msg).toBe("Bad Request")
+        })
+    })
+
```

Comment:
Could we just add a few more tests here please? 
- Status 201, which ignores unnecessary properties
- Status 404, non existent ID, e.g. 0 or 9999
- Status 400, missing required field(s), e.g. no username or body properties